cli/cmd/root.go

```diff
@@ -274,6 +289,45 @@ func registryOverride(image, registry string) string {
 	return strings.Replace(image, defaultDockerRegistry, registry, 1)
 }
 
+// mapRangeSlice creates and populates the target slice of unsigned integers
+// based upon the source slice of string integer(s) and range(s).
+// For example:
+//   []string{"23","25-27"}
+// will result in
+//   []uint{23,25,26,27}
+func mapRangeSlice(target *[]uint, source []string) error {
```

Comment:
I wonder whether it makes sense to extract that into a separate file. Having a `root_test.go` that only tests this functionality seems a bit odd to me. WDYT?

Reply:
I had actually considered a separate file at first, but was worried about starting a utility method "junk drawer."  I do like the idea of creating a `range_slice.go` however which would be similar to how the `pflags` is organized with `string_slice.go` and `uint_slice.go`.  Does `range_slice.go` or `port_range_slice.go` sound better?

Reply:
I think `range_slice.go` sounds good. "Utility method junk drawer" - that is a good one !